app/dashboard/views.py

```diff
@@ -6493,6 +6493,33 @@ def payout_bounty_v1(request, fulfillment_id):
     return JsonResponse(response)
 
 
+@csrf_exempt
+@require_POST
+def payout_tx_forwarder_v1(request, tenant):
```

Comment:
rename this as well to `reverse_proxy_rpc_v1` (or something link that)

Reply:
done